### DISCUSSION TITLE
UMEP-SEBE

### DISCUSSION THREAD

**Discussion by GLauretti** [UMEP-dev/UMEP#267] [Q&A]
Hi,
I'm working on a simulation with SEBE and I was wondering if there is any chance to have the hourly energy on the rooftops.
Can anyone tell me that?
Thank you for the attention
Best regards
Giulia Lauretti

**Comment by biglimp** [comment]
Hi Giulia,
Unfortunately, the code is not written sp that it is easy to retrieve hourly energy. The reason is that the model would be very slow. With that said, recent developments in SOLWEIG where shadows are pre-calculated before model execution have potentially made it possible to change SEBE to output hourly values, but then the model must be re-coded. Maybe sometime in the future I will undergo this task...
